presto-main/src/main/resources/webapp/src/components/QueryDetail.jsx

```diff
@@ -38,24 +37,18 @@ import {
 } from "../utils";
 import {QueryHeader} from "./QueryHeader";
 
-const Table = Reactable.Table,
-    Thead = Reactable.Thead,
-    Th = Reactable.Th,
-    Tr = Reactable.Tr,
-    Td = Reactable.Td;
-
-class TaskList extends React.Component {
```

Comment:
the following changes convert the `TaskList` from a React class to a React function which is a better approach in the current React version.